artemis/src/test/java/tech/pegasys/artemis/EventBusExceptionHandlerTest.java

```diff
@@ -41,6 +46,8 @@
   private final SafeFuture<Throwable> handledException = new SafeFuture<>();
   private final SafeFuture<Throwable> unhandledExceptionFuture = new SafeFuture<>();
 
+  @Mock private Logger log;
```

Comment:
We use the static methods, not the annotations.
```suggestion
  private Logger log = Mockito.mock(Logger.class);
```

typically with `mock` as a static import.